src/main/webapp/login-status.js

```diff
@@ -0,0 +1,25 @@
+// Check if user is logged in a display login status.
+async function getLoginStatus() {
+  const response = await fetch('/login');
+  const loginStatus = await response.json();
+  return loginStatus;
+}
+
+// Set HTML content according to whether user is logged in.
+function displayLoginStatus(loginStatus) {
+  const loginStatusElement = document.getElementById('login-status');
+
+  if (loginStatus.loggedIn) {
+    loginStatusElement.innerHTML =
+    `<a href=${loginStatus.logoutUrl}>Log out</a>`;
+  } else {
+    const loginLink = `<a href=${loginStatus.loginUrl}>Log in</a>`;
+    loginStatusElement.innerHTML = loginLink;
```

Comment:
Is there a reason not to do this in one line, like the logout one does?

Reply:
Nope, fixed